x-pack/plugins/observability_solution/apm/public/components/app/service_inventory/index.tsx

```diff
@@ -319,6 +330,11 @@ export function ServiceInventory() {
             onChangeTableSearchBarVisibility={() => {
               saveSingleSetting(apmEnableServiceInventoryTableSearchBar, !isTableSearchBarEnabled);
             }}
+            measureName="apm.services.table.services.list"
+            onMeasureComplete={
```

Comment:
Are those properties being used? I could have missed something though.

Another point is we usually name properties with `onXXX` when an action needs to be taken. But in this case, there is no action on the `onMeasureComplete`, am I right?

Reply:
good catch, this was experimental I should have cleaned it up

Reply:
fixed here https://github.com/elastic/kibana/pull/180309/commits/e5285410a8b2a4d426269856205bd1023f6be3fb